src/lib/image/IOtiff/IOtiff.cpp

```diff
@@ -345,9 +345,9 @@ readAllTags(TIFF* tif, FrameBuffer& img)
     //  Now the rest of the tags
     //
    
-    for (int i = 0, n = tif->tif_nfields; i < n; i++)
+    for (int fi = 0, nfi = tif->tif_nfields; fi < nfi; fi++)
     {
-        const TIFFFieldInfo* fip = tif->tif_fieldinfo[i];
```

Comment:
Inside TIFF, data was moved around: FieldInfo was merged in TIFFField and the info (at least field_tag) is now accessible via tif_fields.